nautobot-app/{{ cookiecutter.project_slug }}/tasks.py

```diff
@@ -848,6 +848,27 @@ def unittest_coverage(context):
     run_command(context, command)
 
 
+@task(pre=[unittest])
+def unittest_xml_coverage(context):
+    """Produce an XML coverage report that can be ingested into other tools."""
+    produce_xml_cmd = "coverage xml --include '{{ cookiecutter.app_name }}/*' --omit '*/migrations/*,*/tests/*'"
```

Comment:
I wonder if it would be preferable **not** to omit `migrations` (since we should be sure to test data migrations if possible) and `tests` (since it would flag cases of dead test code that never gets called, etc.)

Reply:
Took in changes in latest push